File: Pohon Natal/Python/pohon-natal-i.py
x = int(input('Masukan Tinggi Pohon : '))
for i in range(1, x + 1):
    print(' ' * (x - i) + '*' * i + '*' * (i - 1))
    if(i==x):
        for j in range(1, 3):
            print(' ' * (x-2) + '===')

File: Pohon Natal/C/pohon-natal-i.c
#include <stdio.h>
int main(){
    int x;
    printf("Masukan Tinggi Pohon : ");
    scanf("%x", &x);
    for(int i = 1; i <= x; i++){
        for(int j = x; j >= i; j--){
            printf(" ");
        }
        
        for(int l = 2; l <= i; l++){
            printf("*");
        }
        
        for(int k = 1; k <= i; k++){
                printf("*");
            }
            
        printf("\n");
            
        if(i==x){
            for(int m = 0; m < 3; m++){
                for(int o = 1; o < i; o++){
                printf(" ");
                }
                
                for(int n = 0; n <= 0; n++){
                printf("===");
                }
                
            printf("\n");
            }
        }
    }
    return 0;
}
